lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -155,10 +155,10 @@ export abstract class ClientApplication {
                 // Cache the homepage under ORIGIN_URI to ensure cached hash is processed on homepage
                 this.browserStorage.setItem(this.browserStorage.generateCacheKey(TemporaryCacheKeys.ORIGIN_URI), homepage, CacheSchemaType.TEMPORARY);
                 this.logger.warning("Unable to get valid login request url from cache, redirecting to home page");
-                BrowserUtils.navigateWindow(homepage, true);
+                BrowserUtils.navigateWindow(homepage, this.logger, true);
```

Comment:
We should try to revisit logger so we don't have to pass the handle to utils in the future.